scripts/baseline_imrphenomd_nrtidalv2.py

```
@@ -0,0 +1,333 @@
+            hp_lal, hc_lal = get_lal_waveform(
+                theta_lal,
+                "IMRPhenomD_NRTidalv2",
+                f_l, f_u, df, f_ref,
+                is_tidal=True,
+                is_precessing=False,
+            )
+            hp_lal_list.append(jnp.array(hp_lal))
+            hc_lal_list.append(jnp.array(hc_lal))
+            theta_ripple_list.append(convert_parameters_lal_to_ripple(theta_lal))
+            valid_mask[i] = True
+        except Exception as e:
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)